src/datasources/notifications/entities/notification-devices.entity.db.ts

```diff
@@ -1,26 +1,37 @@
-import { NotificationSubscription } from '@/datasources/notifications/entities/notification-subscription.entity';
+import { NotificationSubscription } from '@/datasources/notifications/entities/notification-subscription.entity.db';
+import { DeviceType } from '@/domain/notifications/v2/entities/device-type.entity';
 import type { UUID } from 'crypto';
 import {
-  Check,
   Column,
   Entity,
   Unique,
   OneToMany,
   PrimaryGeneratedColumn,
 } from 'typeorm';
+import { z } from 'zod';
+
+export const NotificationDeviceSchema = z.object({
+  id: z.number(),
+  device_type: z.nativeEnum(DeviceType),
+  device_uuid: z.string().uuid(),
```

Comment:
We can use the `UuidSchema` here, though we can do that in a follow up.

Reply:
Done
https://github.com/safe-global/safe-client-gateway/pull/2063/commits/5fc9e0bf5ba876172c32f7eb85554bf3e6b488ce